Frameworks/CoreImage/CIContext.mm

```diff
@@ -74,8 +74,7 @@ + (CIContext*)contextWithEAGLContext:(EAGLContext*)eaglContext options:(NSDictio
  @Notes parameter dict is ignored.
 */
 + (CIContext*)contextWithOptions:(NSDictionary*)dict {
-    CIContext* context = [[CIContext alloc] init];
-    return context;
+    return [[CIContext alloc] init];
 }
```

Comment:
Same reason I touched it, but ARC is enabled :)

---
In reply to: [94636365](https://github.com/Microsoft/WinObjC/pull/1625#discussion_r94636365) [](ancestors = 94636365,94636333)